src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java

```diff
@@ -0,0 +1,39 @@
+package com.wikia.webdriver.testcases.globalnavigationtests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.HomePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.globalnav.GlobalNavigationPageObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ExploreWikiaDropdown extends NewTestTemplate {
+
+  private Dimension HUBS_IN_DROPDOWN_RESOLUTION = new Dimension(768, 1024);
+  private Dimension HUBS_OUTSIDE_DROPDOWN_RESOLUTION = new Dimension(1024, 1024);
+  private List<String> EXPECTED_LINKS_BIG_RESOLUTION =
+          Arrays.asList("Top Communities", "Community Central", "START A WIKIA");
+  private List<String> EXPECTED_LINKS_SMALL_RESOLUTION =
+          Arrays.asList("Games", "Movies", "TV", "Top Communities", "Community Central", "START A WIKIA");
+
+  @Test(groups = {"TestExploreWikiaDropdown_001", "TestExploreWikiaDropdown", "GlobalNav"})
```

Comment:
please move `TestExploreWikiaDropdown", "GlobalNav"` groups and put them on class